Lib/fontTools/cffLib/specializer.py

```diff
@@ -220,14 +223,15 @@ def generalizeCommands(commands, ignoreErrors=True):
 			if ignoreErrors:
 				# Store op as data, such that consumers of commands do not have to
 				# deal with incorrect number of arguments.
-				result.append(('', args))
-				result.append(('', [op]))
+				if len(args):
+					result.append(('', args))
+					result.append(('', [op]))
 			else:
 				raise
 	return result
 
 def generalizeProgram(program, **kwargs):
-	return commandsToProgram(generalizeCommands(programToCommands(program), **kwargs))
+	return commandsToProgram(generalizeCommands(programToCommands(program, **kwargs), **kwargs))
```

Comment:
This doesn't work, as generalizeCommands() takes arguments that programToCommands() doesn't take.